### DISCUSSION TITLE
Send button not working well

### DISCUSSION THREAD

**Discussion by Youthleap423** [MessageKit/MessageKit#1685] [Q&A]
When I input the message and tap the send button, it only hide the keyboard and not sending messages.  After keyboard is hidden, tap the send button again, it works.  Please help me why it happens.
This is my code. 

func inputBar(_ inputBar: InputBarAccessoryView, didPressSendButtonWith text: String) {
        actionSendMessage(text)
}

**Comment by lowkeyboard** [comment]
Have you tried reloading data? (i.e: messages.CollectionView.reloadData() )

**Reply by ChanSeulCho** [reply]
same problem... why..
